plugins/kotlin/j2k/shared/tests/testData/newJ2k/issues/kt-824-isDir.kt

```diff
@@ -11,8 +11,6 @@ object Test {
             return false
         }
         val result = true
-        return if (parent.isDirectory) {
-            true
-        } else false
+        return if (parent.isDirectory) true else false
```

Comment:
All curly blocks, formatting and comments of nested elements need to be preserved (and we should make sure to have tests to check that)